wkconnect/backends/wkw/models.py

```diff
@@ -20,7 +25,8 @@
 class Dataset(DatasetInfo):
     organization_name: str
     dataset_name: str
-    dataset_handle: WKDataset = None
+    dataset_handle: WKDataset
+    repo: DatasetRepository
```

Comment:
I assume this repository is a cache? Is there a specific reason for this name? I confused it with https://github.com/scalableminds/webknossos-connect/blob/master/wkconnect/repository.py – maybe this could just be called something with wkw cache?